app/scss/_contact.scss

```diff
@@ -1,65 +1,102 @@
-section[id=contact] {
-  font-family: Arial, sans-serif;
-  width: 100%;
-  background: #EFEFEF;
-  padding: 100px 0;
-  position: relative;
+@mixin border-radius($radius) {
+  -webkit-border-radius: $radius;
+  -moz-border-radius: $radius;
+  -ms-border-radius: $radius;
+  -o-border-radius: $radius;
+  border-radius: $radius;
+}
+@mixin transition($property, $duration, $timing) {
+  -webkit-transition: $property $duration $timing;
+  -moz-transition: $property $duration $timing;
+  -ms-transition: $property $duration $timing;
+  -o-transition: $property $duration $timing;
+  transition: $property $duration $timing;
+}
 
-  &::before {
-    content: '';
-    height: 400px;
-    display: block;
-    background: orange;
-    padding: 30px 0;
-    box-shadow: 0 0 50px -10px #303030;
-  }
+$akai-yellow: #f9a11b;
+$heading-color: #333;
+$text-color: #8a8a8a;
+$bg-color: #f5f5f5;
+$border: 3px solid #c7c7c7;
+
+html, body {
+  font-size: 10px;
+  margin: 0;
+}
+
+.contact {
+  color: $heading-color;
+  width: 100%;
+  background: $bg-color;
+  padding: 5rem 0;
 
   .contact-box {
-    position: absolute;
-    width: 50%;
-    left: 25%;
-    right: 25%;
-    top: 50%;
-    transform: translateY(-50%);
+    margin: 0 auto;
+    max-width: 65rem;
+    width: 80%;
     text-align: center;
-    background: #FFF;
-    padding: 30px 0;
-    border-radius: 2px;
-    box-shadow: 0 20px 80px -30px #303030;
 
-    p {
-      margin: 40px 30px;
+    .contact-heading {
+      font-size: 2.6rem;
+
+      &::after {
+        content: '';
+        background: $akai-yellow;
+        width: 4.5rem;
+        height: .3rem;
+        margin: 2.5rem auto;
+        display: block;
+      }
+    }
+
+    .contact-info {
+      margin: 1.5rem 3rem 4rem;
+      color: $text-color;
+      line-height: 2.6rem;
+      font-size: 1.5rem;
     }
 
-    form {
-      margin: 0 auto 30px;
+    .contact-form {
+      margin: 0 auto 2.5rem;
       display: block;
-      max-width: 350px;
+      max-width: 35rem;
 
-      input, textarea {
-        font-family: Arial, sans-serif;
+      .email, .message {
         box-sizing: border-box;
+        font-family: inherit;
```

Comment:
Zwróć uwagę co mozilla/chrome robią z formularzami (np. kiedy nie wypiszesz poprawnie emaila czy chociażby ::focus) - warto sprawdzić czy i jak można te elementy stylować - myślę, że nie chcemy nic pokazywać przy focusie (ewentualnie ciemniejsza czcionka, nie jak placeholder), a przy niewłaściwym wpisie można to podkreślenie zmienić z szarego na czerwone.